common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -365,6 +365,13 @@ object CometConf {
     .booleanConf
     .createWithDefault(false)
 
+  val COMET_CAST_STRING_TO_TIMESTAMP: ConfigEntry[Boolean] = conf(
```

Comment:
Wondering if we should have this configuration at all? If users turn  this on they could end up with incorrect data?